include/megaapi.h

```diff
@@ -11935,6 +11963,15 @@ class MegaApi
          */
         void setRubbishBinAutopurgePeriod(int days, MegaRequestListener *listener = NULL);
 
+        /**
+         * @brief Returns the id of this device
+         *
+         * You take the ownership of the returned value.
+         *
+         * @return The id of this device
+         */
+        const char* getDeviceId() const;
```

Comment:
This was done only to aid with test code. But I'll let @sergiohs84 decide.

Reply:
Clients will add the ability to set the device name in a near future. Hence, they will need to check their own device id.
Imagine a list of devices for the user to see the last session ts, country... (like in Security in Webclient). The idea is that, instead of anonymous sessions, they map to the device id, and the user attribute that stores the names given to each device, will be used to ease the identification of devices by the user. At some point, the app will need to know which is its own device id.

Reply:
Actually, thinking twice about it, I don't see the need to expose the device id for tests. Instead of calling the generic `getUserAttribute()` and then search for your device id, you could use the dedicated getter `getDeviceName()`, which directly returns the name of your device.
Nevertheless, I think it's not a problem to keep it, and can be beneficial soon. Up to you, I don't have an strong opinion on this one, but I'd keep it.

Reply:
Is there anything that must be changed after this discussion?

Reply:
Ok, no need to change it, let's see how it plays out.  thanks